docs/topics/pagination.txt

```diff
@@ -104,6 +104,20 @@ The view function looks like this::
 
         return render(request, 'list.html', {'contacts': contacts})
 
+The boilerplate code above is also available as utility function
```

Comment:
I think I'd just replace the code above with the new API. I don't see a reason why one would wan to do this manually.